uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/document.js

```diff
@@ -79,13 +86,15 @@ define([
           return deferred.promise;
         },
         getOptions: function () {
-          var deferred = $q.defer();
-          var deferredDocumentType = $q.defer();
+          return $q.all({
+              documentType: this.getDocumentTypes(),
+              documentStatus: this.getDocumentStatus()
+            }).then(function (options) {
```

Comment:
if you're just returning `options` without doing anything with it, then you can skip the `then` altogether